wpuf-functions.php

```diff
@@ -369,6 +373,7 @@ function wpuf_category_checklist( $post_id = 0, $selected_cats = false, $attr =
 
     $categories = (array) get_terms( $tax, array(
```

Comment:
Can't we do this thing from our PRO plugin via a filter? Why a pro feature code should be in the free version?